WordPress/Classes/ViewRelated/Views/List/NotificationsList/AvatarsView.swift

```diff
@@ -0,0 +1,144 @@
+import SwiftUI
+import DesignSystem
+
+struct AvatarsView: View {
+    private enum Constants {
+        static let doubleAvatarHorizontalOffset: CGFloat = 18
+        static let tripleAvatarViewHeight: CGFloat = 44
+    }
+
+    enum Style {
+        case single(URL)
+        case double(URL, URL)
+        case triple(URL, URL, URL)
```

Comment:
nit: I like modeling this as an `enum`, but we can improve this more by adding a convenience init that takes an array `[URL]` and converts it to this enum. This would be simplify usage a little bit.

```swift
// URLs from the Backend
let urls = [...]

/// Convert it to Style
let style = Style(urls: url)
```

Reply:
The reason I used enum was to limit the amount of URLs that can be passed in. Also to unburden this UI component from deciding which index must have precedence in UI hierarchy (Zindex). 

This way the call-site of AvatarsView is responsible for dealing with any raw input like backend response and it doesn't make any assumptions about URLs. That being said, we can definitely add such a helper to the call-site and keep it in a private `AvatarsView` extension.

I don't feel super strong about this. I just wanted to explain my thinking process. Let me know what you think. If you like to have an array convenience init still, I can add it.